docker-compose.yml

```diff
@@ -13,9 +13,11 @@ services:
       - PYTHONUNBUFFERED=1
       - PYTHONASYNCIODEBUG=1
       - ILOOP_TOKEN
-      - ILOOP_API=${ILOOP_API:-iloop-backend:80/api}
+      - ILOOP_API=${ILOOP_API:-http://iloop-backend:80/api}
     networks:
-      - iloop-net
+      iloop-net:
```

Comment:
When the https://github.com/biosustain/iloop/pull/73 gets merged (tomorrow morning), the network will be renamed to just `iloop`.